src/translate_ligo.coffee

```diff
@@ -1,7 +1,11 @@
 module = @
 require "fy/codegen"
 config = require "./config"
+Type = require "type"
 {translate_var_name, spec_id_translate} = require "./translate_var_name"
+{default_var_map_gen} = require "./type_inference/common"
+ti_map = default_var_map_gen()
+ti_map["encodePacked"] = new Type "function2<function<bytes>,function<bytes>>"
```

Comment:
Why not add this value straight to the `default_var_map_gen()`?

Reply:
It's already added as abi.encodePacked, but we need this function by name unpacked on top level, I don't want corrupt original one with fake functions

Reply:
Can you provide an example for that?

Reply:
https://github.com/madfish-solutions/sol2ligo/blob/3aff921591f2dde0f479c973414b3a70849ba90e/test/translate_ligo_fn_api.coffee#L187